### DISCUSSION TITLE
Dark mode for code block? (v3.0.1)

### DISCUSSION THREAD

**Discussion by ajboni** [mdx-editor/editor#433] [Q&A] [answered]
I'm using  dark theme, is any way a set up codemirror theme for the new code block? 
Thanks for your awesome work !

**Comment by petyosi** [comment]
https://github.com/mdx-editor/editor/blob/02b2a82c3605b3527a7aa857fff878fdde54939b/src/examples/toolbar.tsx#L34

**Reply by ajboni** [reply]
hey thanks, maybe i'm confused but I'm asking for the code block, the one that's spawned when the user does triple backtics.

I tried passing ` codeMirrorExtensions: [basicDark]` to ` codeBlockPlugin` but no success.

![image](https://github.com/mdx-editor/editor/assets/12578313/707ab666-3109-46e5-8308-475ff0bcf56c)

**Comment by petyosi** [comment, marked as answer]
The code mirror extensions need to be passed to the`codeMirrorPlugin`. [Check this commit](https://github.com/mdx-editor/editor/commit/912118e406d399034d7fb8c05a9d97a4f156127f).

**Reply by ajboni** [reply]
Of course! Thank you!
